cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
+            cy.viewport(600, 1008);
+            cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+              ({ body }) => {
+                const media = getBlockData('video', body);
+                if (media) {
+                  cy.get('div[class^="StyledVideoContainer"]').then(() => {
+                    cy.get('iframe[class^="i-amphtml-fill-content"]').then(
+                      $iframe => {
+                        cy.wrap($iframe.prop('contentWindow'), {
+                          // `timeout` only applies to the methods chained below.
+                          // `its()` benefits from this, and will wait up to 8s
+                          // for the mediaPlayer instance to become available.
+                          timeout: 8000,
+                        })
+                          .its('embeddedMedia.playerInstances.mediaPlayer')
+                          .invoke('currentTime')
+                          .should('be.gt', 0);
+                      },
+                    );
+                  });
+                }
+              },
+            );
+          });
+
+          it('should render a placeholder image', () => {
+            cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+              ({ body }) => {
+                // `video` blocks can also contain audio.
+                const media = getBlockData('video', body);
+                if (media) {
+                  cy.get('div[class^="StyledVideoContainer"]').within(() => {
+                    cy.get('amp-img')
+                      .should('have.attr', 'src')
+                      .should('not.be.empty');
+                  });
+                }
+              },
+            );
+          });
+          it('should check that media player auto plays', () => {
+            cy.viewport(600, 1008);
+            cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+              ({ body }) => {
+                const media = getBlockData('video', body);
+                if (media) {
```

Comment:
Should we limit this test to media type `video` like we do in the Canonical version?
```
 if (media && media.type === 'video') {
```